core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.spec.ts

```diff
@@ -513,15 +518,21 @@ describe('Question Suggestion Review Modal Controller', function() {
     }));
 
     it('should fetch the rejection message', function() {
-      const responseDict = {
-        messages: [
-          { text: 'Question submitted.' },
-          { text: 'This is a rejection.' }
-        ]
-      };
+      const messages = [
+        { text: 'Question submitted.' },
+        { text: 'This is a rejection.' }
+      ];
+
+      const fetchMessagesAsyncSpy = spyOn(
+        ThreadDataBackendApiService, 'fetchMessagesAsync')
+        .and.returnValue($q.resolve({
+          messages: messages
+        }));
+
+      $scope.init();
+      $rootScope.$apply();
 
-      $httpBackend.expect('GET', '/threadhandler/123').respond(responseDict);
-      $httpBackend.flush();
+      expect(fetchMessagesAsyncSpy).toHaveBeenCalledWith(undefined);
```

Comment:
Assign `$scope.activeSuggestionId` a value so that `threadId` wil not be `undefined`.

Reply:
Done